types/validation.go

```diff
@@ -97,16 +122,35 @@ func (c *Commit) ValidateBasic() error {
 
 func (c *Commit) ValidateWithHeader(proposerPubKey tmcrypto.PubKey, header *Header) error {
 	if err := c.ValidateBasic(); err != nil {
-		return err
+		return NewErrInvalidSignatureFraud(err)
```

Comment:
none of the frauds returned here will be caught in p2p case right? because there is no handler
https://github.com/dymensionxyz/dymint/blob/88ba1fee37359b65a1faf6de32fa94f2ae0c5330/p2p/validator.go#L83-L86